service/src/test/kotlin/app/cash/backfila/development/ServiceHeaderInterceptor.kt

```diff
@@ -0,0 +1,33 @@
+package app.cash.backfila.development
+
+import app.cash.backfila.client.BackfilaApi
+import jakarta.inject.Inject
+import jakarta.inject.Singleton
+import okhttp3.Interceptor
+import okhttp3.Response
+import retrofit2.Invocation
+
+/**
+ * Allows Development services to provide Backfila an authorized name.
+ */
+@Singleton
+class ServiceHeaderInterceptor @Inject constructor(
```

Comment:
This is so dope, we probably should upstream this to Misk in the same jar as FakeCallerAuthenticator.